multiaccounts/settings/database.go

```diff
@@ -144,6 +144,38 @@ INSERT INTO settings (
 		return err
 	}
 
+	// It's enough to set sync clock to any value greater than 0, but not to the current time, cause `CreateSettings` function
+	// is being called for every newly created account (in which case we're good to set current time for the last sync time),
+	// but it's also called for recover account flow, where we need to set data stored on waku (previously backed up data) because
+	// of this we cannot set last sync time to the current time. Also we cannot leave it to 0, cause if account is created, then
```

Comment:
Asked you a question about this, I think I might need more clarification on the `1` case, the first case is clear, the second one maybe we could word it a bit better, but not sure I fully understand